Popup/tests/Popup-specs.js

```diff
@@ -126,81 +281,143 @@ describe('Popup specs', () => {
 		});
 	});
 
-	test('should have `center` class when position prop is set to `center` (default)', () => {
+	test('should close popup on escape', async () => {
+		const handleClose = jest.fn();
 		render(
 			<FloatingLayerController>
-				<Popup open position="center" />
+				<Popup onClose={handleClose} open />
 			</FloatingLayerController>
 		);
 
-		const expected = 'center';
-		const popup = screen.getByRole('alert');
+		userEvent.keyboard('{esc}');
 
-		expect(popup).toHaveClass(expected);
+		await waitFor(() => {
+			expect(handleClose).toHaveBeenCalled();
+		});
 	});
 
-	test('should have `top` class when position prop is set to `top`', () => {
+	test('should apply \'ease-in-out\' class to transition container when noAnimation is false', () => {
 		render(
 			<FloatingLayerController>
-				<Popup open position="top" />
+				<Popup noAnimation={false} open />
```

Comment:
Please remove noAnimation={false}. By defaut the prop has false value and removing it from here does not seem to impact the coverage.